assets/js/components/ClusterDetails/ChecksSelectionNew.test.jsx

```diff
@@ -11,6 +11,29 @@ import { catalogCheckFactory, clusterFactory } from '@lib/test-utils/factories';
 import ChecksSelectionNew from './ChecksSelectionNew';
 
 describe('ClusterDetails ChecksSelectionNew component', () => {
+  it('should render the cluster info box and the catalog container', async () => {
+    const group = faker.animal.cat();
+    const catalog = catalogCheckFactory.buildList(2, { group });
+    const cluster = clusterFactory.build({ provider: 'azure' });
+
+    const initialState = {
+      catalogNew: { loading: false, data: catalog, error: null },
+      clusterChecksSelection: {},
+    };
+    const [statefulChecksSelection] = withState(
+      <ChecksSelectionNew clusterId={cluster.id} cluster={cluster} />,
+      initialState
+    );
+
+    renderWithRouter(statefulChecksSelection);
+
+    const clusterInfoBox = screen.getByTestId('cluster-info-box');
+
+    expect(clusterInfoBox).toBeVisible();
```

Comment:
`expect(screen.getByText('Provider')).toBeVisible()`

Reply:
Thanks for the suggestions! Adjusted to the following
```js
expect(screen.getByText('Provider')).toBeVisible();
expect(screen.getByText('Azure')).toBeVisible();
```
and in order to assert that the checks catalog is there, I also traded `expect(screen.getByTestId('catalog-container')).toBeVisible();` for
```js
expect(
  screen.getByRole('button', { name: 'Select Checks for Execution' })
).toBeVisible();
```
Does it work?